feature/platform/tests/optics_power_and_bias_current_test/README.md

```diff
@@ -1,49 +1,102 @@
-# gNMI-1.13: Telemetry: Optics Power and Bias Current
+# gNMI-1.13: Optics Telemetry, Instant, threshold, and miscellaneous static info
```

Comment:
Would a different test check for min/max/avg values?

Reply:
This is for gray optics only. Plan is to request min/max/avg for ZR